src/list.h

```diff
@@ -177,6 +179,14 @@ ListEntry *list_next(ListEntry *listentry);
 ListValue list_data(ListEntry *listentry);
 
 /**
+ * Set the value at a list entry. If listentry is NULL nothing is done.
+ *
+ * @param listentry 	Pointer to the list entry.
+ * @param value			The value to set.
+ */
+void list_set(ListEntry *listentry, ListValue value);
```

Comment:
Can we name it `list_set_data` to match `list_data`, and also make clear that 'set' is a verb and not referring to the 'set' data type?